ui/desktop/src/App.tsx

```diff
@@ -577,99 +577,48 @@ export default function App() {
+                    />
                   }
                 />
                 <Route
                   path="pair"
                   element={
-                    <ProviderGuard>
-                      <PairRouteWrapper
-                        chat={chat}
-                        setChat={setChat}
-                        agentState={agentState}
-                        loadCurrentChat={loadCurrentChat}
-                        setFatalError={setFatalError}
-                        setAgentWaitingMessage={setAgentWaitingMessage}
-                        setIsGoosehintsModalOpen={setIsGoosehintsModalOpen}
-                      />
-                    </ProviderGuard>
-                  }
-                />
-                <Route
-                  path="settings"
-                  element={
-                    <ProviderGuard>
-                      <SettingsRoute />
-                    </ProviderGuard>
-                  }
-                />
-                <Route
-                  path="extensions"
-                  element={
-                    <ProviderGuard>
-                      <ExtensionsRoute />
-                    </ProviderGuard>
-                  }
-                />
-                <Route
-                  path="sessions"
-                  element={
-                    <ProviderGuard>
-                      <SessionsRoute />
-                    </ProviderGuard>
-                  }
-                />
-                <Route
-                  path="schedules"
-                  element={
-                    <ProviderGuard>
-                      <SchedulesRoute />
-                    </ProviderGuard>
-                  }
-                />
-                <Route
-                  path="recipes"
-                  element={
-                    <ProviderGuard>
-                      <RecipesRoute resetChat={resetChat} />
-                    </ProviderGuard>
-                  }
-                />
-                <Route
-                  path="recipe-editor"
-                  element={
-                    <ProviderGuard>
-                      <RecipeEditorRoute />
-                    </ProviderGuard>
-                  }
-                />
-                <Route
-                  path="permission"
-                  element={
-                    <ProviderGuard>
-                      <PermissionRoute />
-                    </ProviderGuard>
+                    <PairRouteWrapper
+                      chat={chat}
+                      setChat={setChat}
+                      agentState={agentState}
+                      loadCurrentChat={loadCurrentChat}
+                      setFatalError={setFatalError}
+                      setAgentWaitingMessage={setAgentWaitingMessage}
+                      setIsGoosehintsModalOpen={setIsGoosehintsModalOpen}
+                    />
                   }
                 />
+                <Route path="settings" element={<SettingsRoute />} />
```

Comment:
❗

Reply:
oh yeah, to explain this -- in the onboarding, with an empty config, we loaded the onboarding UI but with a non-functional sidebar present. Wrapping the main appcontent solves that because the sidebar is part of it, and it also just makes more sense than wrapping each individual route anyway IMO